x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -206,27 +213,24 @@ specifications, and the list of items gives the numbers of the values governed b
+
+* If `N406` is present, then `N405` is required (`{ type: C, items: [6, 5] }`)
 
+* If `N407` is present, then `N404` is required (`{ type: C, items: [7, 4] }`)
 
 == Modifying Schemas
 
 Due to the differences between standard X12 and HIPAA schemas the use of overlay schemas to modify a base
-definition is not supported for HIPAA. Instead, the recommended approach to use for modifications is to
-extract the supplied HIPAA schema from inside the `x12-schemas-2.0.0.jar`, that is found in the standard MuleSoft enterprise Maven repositories (under group ID com.mulesoft.connectors).
-You can copy a message structure schema from this JAR (that contains both standard X12 and HIPAA schemas) and modify the extracted schema to use
-directly. Unlike X12 schemas, which as supplied use a base set of segment, composite, and element
-definitions, the HIPAA schemas are self-contained. This makes it easy to make changes to the schema
-without needing to work with multiple files.
+definition is not supported for HIPAA. Instead, for modifications extract the supplied HIPAA schema from inside the `x12-schemas-2.0.0.jar`. This is found in the standard MuleSoft enterprise Maven repositories under group ID com.mulesoft.connectors.
+You can copy a message structure schema from this JAR (it contains both standard X12 and HIPAA schemas) and modify the extracted schema to use it directly. Unlike X12 schemas, that use a base set of segment, composite, and element definitions, the HIPAA schemas are self-contained. This makes it easy to make changes to the schema without needing to work with multiple files.
 
 == About Recommended Types of Testing from WEDI
 
 * Type 1: EDI syntax integrity testing – Testing of the EDI file for valid segments, segment order, element attributes, testing for numeric values in numeric data elements, validation of X12 or NCPDP syntax, and compliance with X12 and NCPDP rules.
 This validates the basic syntactical integrity of the EDI submission.
 * Type 2: HIPAA syntactical requirement testing – Testing for HIPAA Implementation Guide-specific syntax requirements, such as limits on repeat counts, used and not used qualifiers, codes, elements, and segments. Also included in this type is testing for HIPAA required or intra-segment situational data elements, testing for non-medical code sets as laid out in the WEDI SNIP implementation guide, and values and codes noted in the WEDI SNIP implementation guide using an X12 code list or table.
-** As the connector cannot determine a course of action for intra-segment situational data elements, intra-segment situational data elements aren't covered by the X12 EDI connector. Users need to set a validation logic outside of the connector.
+** As the connector cannot determine a course of action for intra-segment situational data elements, intra-segment situational data elements aren't covered by the X12 connector. Users need to set a validation logic outside of the connector.
```

Comment:
This should not be a bullet. It should be a bullet continue.

Reply:
Fixed.